test/cypress/integration/Dashboards/Components/DataGrid/DataGridComponent.cy.js

```diff
@@ -81,4 +81,32 @@ describe('layout resize on window changes', () => {
         cy.get('.highcharts-legend-item').eq(0).click();
         cy.get('.highcharts-datagrid-column-headers').children().should('have.length', 2);
     });
+
+    it('DataGridComponent should not throw error when dragging point on chart.', () => {
+        cy.chart().then(chart => {
```

Comment:
Why don't you use the `board.mountedComponents` instead of the `chart` you get there?

Reply:
Nice catch! thanks!